dcat/config.js

```diff
@@ -1,7 +1,9 @@
 var respecConfig = {
 //	preProcess: [dfn_index],
 //    subtitle: "Version 2",
-    specStatus: "ED",
+    specStatus: "REC",
```

Comment:
As suggested by Andrea, we keep the ED status 
```suggestion
    specStatus: "ED",
```